### PR TITLE
update-strategies: use mask and stop

### PR DESCRIPTION
**Pull request by philips**
The best way of accomplishing this is to mask and stop. Also, add docs for people who may wish to do this manually.

### CODE REVIEW

**Comment by philips** [comment]
cc @crawford

File: os/update-strategies.md [hunk 1]
```diff
@@ -89,8 +89,14 @@ coreos:
       command: stop
     - name: locksmithd.service
       command: stop
+    - name: update-engine.service
+      command: mask
+    - name: locksmithd.service
+      command: mask
 ```
 
+Alternatively, to manually mask update-engine run `systemctl mask update-engine.service`. To reverse this remove the stanza from the cloud-config and run `systemctl unmask update-engine.service`
```
Comment (crawford) [inline os/update-strategies.md]: "remove the stanza from the cloud-config" This is not possible on many platforms. We should stop suggesting that cloud-init be used for configuration management. It's really only good for static configuration.

File: os/update-strategies.md [hunk 2]
```diff
@@ -89,8 +89,14 @@ coreos:
       command: stop
     - name: locksmithd.service
       command: stop
+    - name: update-engine.service
+      command: mask
+    - name: locksmithd.service
+      command: mask
```
Comment (crawford) [inline os/update-strategies.md]: Why mask it? cloud-init already stops it.

File: os/update-strategies.md [hunk 3]
```diff
@@ -89,8 +89,14 @@ coreos:
       command: stop
     - name: locksmithd.service
       command: stop
+    - name: update-engine.service
+      command: mask
```
Comment (crawford) [inline os/update-strategies.md]: If this is needed, we should add it to cloud-init.

**Comment by ElijahCaine** [comment]
@philips I'm cleaning up some old PRs.  Any thoughts on this?  If you think it should be merged we should address @crawford's feedback, otherwise I can close this.

**Comment by crawford** [comment]
This is probably fine. coreos-cloudinit is garbage and we can tell users to switch to Ignition when they come to the same realization.

**Comment by ElijahCaine** [comment]
I rebased this PR to get rid of the conflicts.
@philips @joshix Think this should be merged after @crawford's last comment?

**Pull request closed without merging**